lib/wrappers/postgres.nim

```diff
@@ -116,7 +116,9 @@ type
     PQTRANS_UNKNOWN
   PPGVerbosity* = ptr PGVerbosity
   PGVerbosity* = enum
-    PQERRORS_TERSE, PQERRORS_DEFAULT, PQERRORS_VERBOSE
+    PQERRORS_TERSE, PQERRORS_DEFAULT, PQERRORS_VERBOSE, PQERRORS_SQLSTATE
+  PGContextVisibility* = enum
```

Comment:
* where is this used?
* open question: should enum's defined here be importc enums? in particular so that they're guaranteed to stay in sync with C declaration; see how other wrapper libraries define enums; the enum type also (cint?) may need to be specified

Reply:
> where is this used?
In the `PGContextVisibility` and `PQresultVerboseErrorMessage` which not imported now.

Reply:
Ok. I'll remove it due not used now.